meilisearch-lib/src/index_resolver/mod.rs

```diff
@@ -493,17 +411,23 @@ mod test {
     use nelson::Mocker;
     use proptest::prelude::*;
 
-    use crate::index::{
-        error::{IndexError, Result as IndexResult},
-        Checked, IndexMeta, IndexStats, Settings,
+    use crate::{
+        index::{
+            error::{IndexError, Result as IndexResult},
+            Checked, IndexMeta, IndexStats, Settings,
+        },
+        tasks::{batch::Batch, BatchHandler},
     };
     use index_store::MockIndexStore;
     use meta_store::MockIndexMetaStore;
 
+    // TODO: ignoring this test, it has become too complex to maintain, and rather implement
+    // handler logic test.
```

Comment:
Could you create a PR to help us keep track of that? With a permanent link to the associated code part?

Reply:
This is part of my next PR, I need to create a mocker for IndexResolver and work from there. I will remove this test in it :)